scss/components/dialog.scss

```diff
@@ -0,0 +1,121 @@
+@import "./../settings";
+@import "./../mixins";
+@import "./../functions";
+@import "./../icons/mixins";
+
+/*!
+.fd-dialog
+    .fd-dialog__header
+        .fd-form__title
+    .fd-form__body
+    .fd-form__footer-items
+      .fd-dialog__button-primary
+      .fd-dialog__button-secondary
```

Comment:
This does not match the structure of the component. The form items are not part of the component, the footer-items element is no present nor are the button containers.

This block is intended to illustrate the elements and modifiers available.